packages/playwright/src/runner/runner.ts

```diff
@@ -70,6 +71,8 @@ export class Runner {
     const config = this._config;
     const listOnly = config.cliListOnly;
 
+    addGitCommitInfoPlugin(config);
```

Comment:
We'll also need similar call in `testServer.ts`, so that it works in UI mode and with the extension. Let's add a UI mode test for the property.

Reply:
Added a similar call of `addGitCommitInfoPlugin()` in `testServer.ts`. 
I'm not sure how to test it in ui mode, looks like ui mode does not show `config.metadata` (although it shows general metadata):

<img width="499" alt="image" src="https://github.com/user-attachments/assets/329d07ad-e0dd-4baa-9796-93778bf86501" />

I've prepared a test file, could you have a look on `ui-mode-metadata.spec.ts` below.

Reply:
I meant to check that the report files produced when running tests from the UI mode (and VSCode extension, that also goes though the test server) still contain the metadata. I don't think we want to show the metadata in UI mode itself.